salt/modules/napalm_network.py

```diff
@@ -1547,18 +1547,12 @@ def load_config(filename=None,
 
 
 @salt.utils.napalm.proxy_napalm_wrap
-def load_template(template_name,
+def load_template(template_name=None,
                   template_source=None,
-                  template_hash=None,
```

Comment:
I do not like removing the template_hash.  We should probably still keep this, even though cp.get_template does not support giving it a hash.  We should be verifying the hash manually as part of the load_template process.